OrbitClientServices/ProcessManager.cpp

```diff
@@ -57,8 +57,7 @@ class ProcessManagerImpl final : public ProcessManager {
   bool shutdown_initiated_;
 
   mutable absl::Mutex mutex_;
```

Comment:
Nit: I'd prefer to have this named in a less generic way to indicate what it's supposed to be guarding.

Reply:
done.